### DISCUSSION TITLE
Issues generating and using the TUNING FILE for ALSC with Picamera2 and Libcamera

### DISCUSSION THREAD

**Discussion by harshilsheth0412** [raspberrypi/picamera2#304] [Q&A]
Hello folks, 

I am trying to achieve Lens Shading Correction for an imx219 camera on Picamera2. 

I referred to chapter 6 (Camera Tuning) - https://datasheets.raspberrypi.com/camera/raspberry-pi-camera-guide.pdf  and followed the steps to generate a tuning file using ctt.py and two .dng images as input. 

I generated a tuning file without errors. But when I load the tuning file via Picamera2 api or Libcamera-hello --tuning-file input, I get a segmentation fault error. Seems like it has something to do with the tuning file format generated by ctt.py. Am I missing something here? 

I've attached my output logs here: 

[Seg_fault.txt](https://github.com/raspberrypi/picamera2/files/9522715/Seg_fault.txt)

**Comment by davidplowman** [comment]
Hi, thanks for posting this. I see you've also posted this as a bug over at libcamera-apps and there's already been some discussion there. I agree that's probably a better place for it so we'll post any updates there rather than here. Thanks!
